Menu activity

VicPeralta opened the pull request:
#Mobile Menu functionality

We make sure there are no linter errors.
We used the correct GitHub flow.
Documented the repo in a professional way.

This pull request implements the mobile menu functionality:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- Additionally, we made the toolbar fix, so it is easier for the user to navigate the page.

VicPeralta committed:
Add menu options, formating and basic functionality

modified:   index.html
new file:   menu.css
new file:   menu.js

estebanmual committed:
Add menu option funtionality and give navbar a fixed position

Cathella commented on menu.js:
```diff
@@ -0,0 +1,40 @@
+
```
- [x] Hello @VicPeralta, you have indeed done a great job with the mobile menu feature and I have checked and most of the requirements have been met 💪🏻! However, since you are adding javaScript to this project, I suggest that you set up both JavaScript and HTML&CSS linters in the project repository. It will help you have quality code for the javaScript file too.

VicPeralta replied:
Thank you, I forgot to included it.
By the way, I didn't receive an email with the changes required and I couldn't find the comments but thankfully in the end, I found it.

Cathella replied:
It was actually an error on my end 🥲 . I am sorry!

Cathella reviewed (changes requested):
## 🟨 Status: Changes Required 🛑

### Hi @VicPeralta 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻 

✅ No linter errors.
✅ You used correct Github Flow.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

VicPeralta committed:
Add linter for js

new file:   .eslintrc.json
modified:   .github/workflows/linters.yml
modified:   menu.js
modified:   package-lock.json
modified:   package.json

Cathella reviewed (approved):
## 🟢 Status: Approved 🔥 💪🏻 

### Hi @VicPeralta 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
